.github/workflows/ci-tests.yml

```diff
@@ -114,13 +140,20 @@ jobs:
     strategy:
       matrix:
         dbt-version: [ 'dbt018', 'dbt019', 'dbt020', 'dbt021' ]
+    env:
+      PYTHON_VERSION: '3.8'
     name: DBT Plugin ${{ matrix.dbt-version }} tests
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: '3.8'
+        python-version: ${{ env.PYTHON_VERSION }}
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: ${{ env.pythonLocation }}
+        key: ${{ env.PYTHON_VERSION }}-${{ runner.os }}-build-${{ matrix.dbt-version }}-${{ hashFiles('tox.ini') }}
```

Comment:
Include `${{ hashFiles('setup.py') }}` and `${{ hashFiles('requirements_dev.txt') }}` and `${{ hashFiles('plugins/sqlfluff-templater-dbt/setup.py') }}` in hash key